lib/app_profiler/viewer/speedscope_remote_viewer/base_middleware.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
+gem "rails-html-sanitizer", ">= 1.6.0"
```

Comment:
@flavorjones To enforce the version dependency, I had to add a peer dependency here. (There is no dependency to that gem in the gemspec.)